infra-provider/src/main/java/com/homeaway/streamplatform/streamregistry/provider/InfraManager.java

```diff
@@ -30,15 +30,15 @@
  *     <li><code>env</code> - Environment, for example staging or production</li>
  *     <li><code>hint</code> - If there are multiple clusters in a vpc, the specific classifier of the cluster. E.g: metrics, logging, etc.</li>
  *     <li><code>type</code> - If there are multiple clusters in a vpc for a type of hint, the specific type of the cluster. E.g. metrics-aggregate or logging-mirror.</li>
- * </ul></p>
```

Comment:
End paragraph is still needed.

Reply:
Looks like it's not. The javadoc check was throwing an error. Not exactly HTML like. Here's the official doc - https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format.